src/MoBi.Core/Commands/AddTagConditionCommandBase.cs

```diff
@@ -22,17 +22,17 @@ protected AddTagConditionCommandBase(string tag, T taggedObject, IBuildingBlock
       protected override void ExecuteWith(IMoBiContext context)
```

Comment:
I would rename the file from AddTagConditionCommandBase to AddTagConditionCommand